components/landing/topCollections/TopCollections.vue

```diff
@@ -1,8 +1,10 @@
 <template>
   <div>
-    <div class="flex justify-between mobile">
-      <div class="title is-2">{{ $t('general.topCollectionsHeading') }}</div>
-      <div class="top-collection-controls buttons items-start pt-2">
+    <div class="flex justify-between mobile pb-9 md:pb-10">
+      <div class="title is-2 md:!mb-0">
```

Comment:
`title is-2` will be remove (#9013) please could you use tailwind helper instead?

Reply:
done